universal-application-tool-0.0.1/test/views/questiontypes/IDQuestionRendererTest.java

```diff
@@ -0,0 +1,82 @@
+package views.questiontypes;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.google.common.collect.ImmutableSet;
+import j2html.tags.Tag;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.OptionalLong;
+import org.junit.Before;
+import org.junit.Test;
+import play.i18n.Lang;
+import play.i18n.Messages;
+import play.i18n.MessagesApi;
+import repository.WithPostgresContainer;
+import services.LocalizedStrings;
+import services.applicant.ApplicantData;
+import services.applicant.question.ApplicantQuestion;
+import services.question.types.IDQuestionDefinition;
+import services.question.types.IDQuestionDefinition.IDValidationPredicates;
+import support.QuestionAnswerer;
+
+public class IDQuestionRendererTest extends WithPostgresContainer {
+  private static final IDQuestionDefinition ID_QUESTION_DEFINITION =
+      new IDQuestionDefinition(
+          OptionalLong.of(1),
+          "question name",
+          Optional.empty(),
+          "description",
+          LocalizedStrings.of(Locale.US, "question?"),
+          LocalizedStrings.of(Locale.US, "help text"),
+          IDValidationPredicates.create(2, 3));
+
+  private final ApplicantData applicantData = new ApplicantData();
+
+  private ApplicantQuestion question;
+  private Messages messages;
+  private ApplicantQuestionRendererParams params;
+  private IDQuestionRenderer renderer;
+
+  @Before
+  public void setUp() {
+    question = new ApplicantQuestion(ID_QUESTION_DEFINITION, applicantData, Optional.empty());
+    messages = instanceOf(MessagesApi.class).preferred(ImmutableSet.of(Lang.defaultLang()));
+    params = ApplicantQuestionRendererParams.sample(messages);
+    renderer = new IDQuestionRenderer(question);
+  }
+
+  @Test
+  public void render_withoutQuestionErrors() {
+    Tag result = renderer.render(params);
+
+    assertThat(result.render()).doesNotContain("Must contain at");
```

Comment:
Maybe just look for "Must contain" to also match on the "Must contain only numbers." error?